plugins/afl-scoreboard/scroll_display.py

```
@@ -1,694 +1,347 @@
+                icon_path = separator_dir / filename
+                if icon_path.exists():
+                    try:
+                        icon = Image.open(icon_path).convert('RGBA')
+                        # Scale to fit display height if needed
+                        if icon.height > self.display_height - 4:
+                            scale = (self.display_height - 4) / icon.height
+                            new_width = int(icon.width * scale)
+                            new_height = int(icon.height * scale)
+                            icon = icon.resize((new_width, new_height), Image.LANCZOS)
+                        self._separator_icons[league_key] = icon
+                        self.logger.debug(f"Loaded {LEAGUE_NAMES[league_key]} separator icon: {icon.size}")
```

Ruff (0.16.0)

[error] Undefined name `LEAGUE_NAMES`

(F821)